### ISSUE TITLE
[FR] Display command log window

### ISSUE THREAD

**Issue by yantar92** [ir33k/tb-keycast#7]
Seeing the last command is indeed nice, but easy to miss at times. It would be useful to add an option to display recent commands in a side window slot in Emacs.

**Comment by ir33k** [comment]
There is `C-h l` (`view-lossage`) that can be use to see history of commands but it's a static buffer.  To refresh it you have to press `g`.

With current code of `tb-keycast` it could be quite easy to just create `*tb-keycast*` buffer and append each last value of `tb-keycast--status` without the repeat key number or something like that.

**Comment by yantar92** [comment]
> There is `C-h l` (`view-lossage`) that can be use to see history of commands but it's a static buffer.  To refresh it you have to press `g`.

Yes, but it does not compress repeated commands with xN and does not have
a nice-looking font-locking.

**Comment by ir33k** [comment]
True.  I hesitate with adding more stuff to this package.  This feature could be implemented as separate thing that depends on this package.

To make it possible `tb-keycast--status` has to be separated to 2 different variables.  First with just the data in forma of the string and second with all the faces, text properties and margins that is used to put status in `tab-bar`.

With that it would be possible to just create another package and, bind to the same hook and use value of status var to gather and print whole history list.
